CONTRIBUTING.md

```diff
@@ -37,10 +37,10 @@ If you have an idea for an improvement to the code or the docs, we encourage you
 
```

Comment:
I think we should copy MUI X here: https://github.com/mui/mui-x/blob/next/CONTRIBUTING.md